src/Axe/Data/AxeDbDeployment.cs

```diff
@@ -262,6 +262,44 @@ public static class AxeDbDeployment
 
             context.Add(task);
 
+            #region Code Blocks
+
+            var codeBlocks = new CodeBlock[]
+            {
+                new CodeBlock {
+                    Code = @"
+                            using System;
```

Comment:
it is ok, but a string with `@` keeps all 28 spaces at the start of each line, and this code displayed on page (e.g. in a textarea), with unexpected indent